src/search.c

```diff
@@ -613,12 +613,11 @@ on_widget_key_pressed_set_focus(GtkWidget *widget, GdkEventKey *event, gpointer
 static void create_replace_dialog(void)
 {
 	GtkWidget *label_find, *label_replace,
-		*check_close, *button, *rbox, *fbox, *vbox, *exp, *bbox;
+		*check_close, *button, *fbox, *vbox, *exp, *bbox;
 	GtkSizeGroup *label_size;
 
-	replace_dlg.dialog = gtk_dialog_new_with_buttons(_("Replace"),
-		GTK_WINDOW(main_widgets.window), GTK_DIALOG_DESTROY_WITH_PARENT,
-		GTK_STOCK_CLOSE, GTK_RESPONSE_CANCEL, NULL);
+	replace_dlg.dialog = gtk_dialog_new();
```

Comment:
As @elextr mentionedin the PR comments, this doesn't set the transient parent anymore, I believe you can set it with (`gtk_window_set_transient_for()`)[https://developer.gnome.org/gtk3/stable/GtkWindow.html#gtk-window-set-transient-for].